### PR TITLE
feat: download album

### PR DESCRIPTION
**Pull request by y-lohse**

### CODE REVIEW

**Comment by y-lohse** [comment]
[(1) [PHO-31] Télécharger un album depuis la vue album](https://trello.com/c/pgPSwbEN/283-1-pho-31-t%C3%A9l%C3%A9charger-un-album-depuis-la-vue-album)

File: src/ducks/albums/components/AlbumToolbar.jsx [hunk 1]
```diff
@@ -86,7 +92,8 @@ export const mapDispatchToProps = (dispatch, ownProps) => ({
       .then(() => ownProps.router.replace('albums'))
       .then(() => Alerter.success('Albums.remove_album.success', {name: album.name}))
       .catch(() => Alerter.error('Albums.remove_album.error.generic'))
-  )
+  ),
+  downloadAlbum: downloadAlbum
```
Comment (goldoraf) [inline src/ducks/albums/components/AlbumToolbar.jsx]: As downloadAlbum is not an action creator (no dispatch), it isn't necessary to declare it here, you could just call downloadAlbum directly in the event handler.

File: src/ducks/albums/components/AlbumToolbar.jsx [hunk 2]
```diff
@@ -31,7 +31,8 @@ class AlbumToolbar extends Component {
   }
 
   render () {
-    const { t, album, disabled = false, deleteAlbum, selectItems, shareAlbum, onRename } = this.props
+    const { t, album, photos, disabled = false, deleteAlbum, downloadAlbum, selectItems, shareAlbum, onRename } = this.props
+    console.log(photos)
```
Comment (goldoraf) [inline src/ducks/albums/components/AlbumToolbar.jsx]: :boom:

Comment (y-lohse) [inline src/ducks/albums/components/AlbumToolbar.jsx]: wooops.

**Pull request merged**